classes/game.rb

```diff
@@ -8,4 +9,10 @@ def initialize(publish_date, multiplayer, last_played_at)
     @multiplayer = multiplayer
     @last_played_at = last_played_at
   end
+
+  def can_be_archived?
+    game_date = DateTime.parse(@last_played_at).to_date
+    years = (Date.today.year - game_date.year) > 2
+    super ? super && years : false
```

Comment:
Here you are required to: 
- should override the method from the parent class.
- should return true if parent's method returns true AND if last_played_at is older than 2 years.
- otherwise, it should return false.

**Speaking of overriding you can use the `super` keyword to call the parent method.**
E.g: `super && years`